cmd/start.go

```diff
@@ -73,6 +75,8 @@ func init() {
 	viperBindFlag("helm.namespace", startCmd.Flags().Lookup("helm-ns"))
 	viperBindFlag("helm.releasePrefix", startCmd.Flags().Lookup("helm-prefix"))
 	viperBindFlag("helm.tiller", startCmd.Flags().Lookup("helm-tiller"))
+	viperBindFlag("helm.wait", startCmd.Flags().Lookup("helm-wait"))
+	viperBindFlag("helm.wait_timeout", startCmd.Flags().Lookup("helm-wait-timeout"))
```

Comment:
Above we use camelCasing, so probably want waitTimeout

Reply:
We use snake casing in the server options, which is why I used it here.

Reply:
These should be camelCase. We should update the ones below to match. Switching between is bad practice and in general go recommends not using underscores.